apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -64,9 +74,9 @@ export function HeaderNav({}: Props) {
     }
 
     return title;
-  };
+  }, [themeStatus]);
 
-  const Icon = () => {
+  const Icon = useCallback(() => {
```

Comment:
To avoid unnecessary re-renders as it is a dependency of the `useEffect` implementation below.